rust/catalyst-voting/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 //! Voting primitives which are used among Catalyst ecosystem.
 
+#![allow(clippy::arithmetic_side_effects)]
```

Comment:
There is a lot of arithmetic in `catalyst-voting/crypto` and I'm not sure that it should be changed. It needs to be carefully evaluated if we want to proceed with applying the lint here.

Reply:
LGTM, but might need to ask @Mr-Leshiy about this

Reply:
yes, disable it for now. I will take a look into this linting errors later